erpnext/buying/doctype/purchase_common/purchase_common.js

```diff
@@ -180,3 +180,78 @@ erpnext.buying.get_default_bom = function(frm) {
 		}
 	});
 }
+
+frappe.ui.form.on(cur_frm.doctype, "get_items_from_product_bundle", function(frm) {
```

Comment:
This is an anti-pattern.

Use:

`erpnext.buying.get_items_from_product_bundle = function(frm) {}`

and bind it in BuyingController instead.